### PR TITLE
Settings menu

### PR DESCRIPTION
**Pull request by stezhu-ux**

### CODE REVIEW

File: Menus/MainMenu/MainMenu.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ public void _on_start_button_pressed()
 	// Enter Settings scene.
 	public void _on_settings_button_pressed()
 	{
-		this.GetRoot().SetScene(Scene.Settings);
+	GetTree().ChangeSceneToFile("res://Menus/Settings/settings.tscn");
```
Comment (jamesp6784) [inline Menus/MainMenu/MainMenu.cs:22]:
```suggestion
		this.GetRoot().SetScene(Scene.Settings);
```

File: Menus/Settings/fps.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+using Godot;
+using System;
+
+public partial class fps : Label
```
Comment (jamesp6784) [inline Menus/Settings/fps.cs:4]: Naming of class and file should be capitalized

File: project.godot [hunk 3]
```diff
@@ -11,18 +11,37 @@ config_version=5
 [application]
 
 config/name="BulletHell"
-run/main_scene="res://Core/Main.tscn"
+run/main_scene="res://Menus/MainMenu/MainMenu.tscn"
```
Comment (jamesp6784) [inline project.godot:14]:
```suggestion
run/main_scene="res://Core/Main.tscn"
```

File: default_bus_layout.tres
Comment (jamesp6784) [inline default_bus_layout.tres:1]: Delete this as unused

File: Menus/Settings/Settings.cs
Comment (jamesp6784) [inline Menus/Settings/Settings.cs:1]: Would probably be good to lint / format code

**Review by jamesp6784 — APPROVED** [review]

**Review by HusseinAljanaby — APPROVED** [review]

**Pull request merged**